diagnostics-extension/src/__tests__/controllers/telemetry.spec.ts

```diff
@@ -77,37 +77,77 @@ describe('should return correct telemetry data', () => {
   const testCases: {
     name: string;
     infoType: TelemetryInfoType;
-    expectedResult: TelemetryInfoUnion;
+    expexted: Promise<TelemetryInfoUnion>;
```

Comment:
Instead of having a promise here, and awaiting it at line 160, can we just await within the object?

This feels more intuitive.

So we will have definitions:
```
const testCases: {
    name: string;
    infoType: TelemetryInfoType;
    expected: TelemetryInfoUnion;
}
```

and testcases:
```
{
      name: `audio`,
      infoType: TelemetryInfoType.AUDIO,
      expected: await fakeData.getAudioInfo(),
    }
```

Reply:
I don't think it can work like that. So instead, I created some fake data in file <code>fake_telemetry.data.ts</code> and used them as the expected results in the unit tests.